lib/symbols.js

```diff
@@ -27,10 +27,9 @@ const keys = {
   kSchemaVisited: Symbol('fastify.schemas.visited'),
   // Request
   kRequest: Symbol('fastify.Request'),
-  kRequestValidateFns: Symbol('fastify.request.cache.validateFns'),
   kRequestPayloadStream: Symbol('fastify.RequestPayloadStream'),
   kRequestAcceptVersion: Symbol('fastify.RequestAcceptVersion'),
-  kRequestValidateWeakMap: Symbol('fastify.request.cache.validators'),
+  kRequestValidateWeakMap: Symbol('fastify.request.cache.validateFns'),
```

Comment:
Can we rename it?

```suggestion
  kRequestCacheValidateFns: Symbol('fastify.request.cache.validateFns'),
```

Of course also need to fix the imports if we rename it.

Reply:
Good one, I rename both ones to be name consistent 👍